Payments/Operation/04_ManageCancellationsChargebacks.md

```diff
@@ -0,0 +1,89 @@
+
+2. If desired, select the cancellation and chargeback type you want to check. The following types are available:   
+    - **Cancel**   
+      Depending on the payment service provider, an appropriately permitted person has cancelled the authorization. 
+    - **Chargeback**   
+      The customer has withdrawn a capture.
+    - **Chargeback Reversal**   
+      The customer has terminated his or her withdrawing of the authorization.
+    - **Second Chargeback**   
+      Again, the customer has withdrawn the authorization.
+
+3. Check the cancellation and chargeback transaction status in the *Status* column. The status displays the current stage in the payment process. You can use the status ID prefixed below to filter the list. The following statuses are available:   
+    - **1 - Preparing**  
+        The transaction has been created, but is not yet transferred.
+    - **2 - Unconfirmed**   
+        A refund is reserved but not yet released by an Actindo user. 
+    - **3 - Success**  
+        Actindo has got the response from the service payment provider, that the transfer was successfully.
+    - **4 - Failure**   
+        The payment service provider has received the request, but it cannot process it. If available for the connection, the status information field provides further information.
+    - **5 - Error**   
+       Errors have been occurred during transfer.
+    - **6 - Void**   
+       The transaction has been voided, see [Void an authorization transaction](./01_ManageAuthorizations.md#void-an-authorization-transaction).
+    
+2. Click on a transaction to see the details.   
+    The sub-tabs of the transaction are opened.   
+
+    *Payments > Cancellations and chargebacks> Tab LIST > Select a cancellation and chargeback transaction*  
+
+    ![Cancellations and chargebacks attributes and logs](../../Assets/Screenshots/Payments/CancellationsChargebacks/CheckCancellationsChargebacks.png "[Cancellations and chargebacks attributes and logs]")
+
+3. Check the attributes and logs of the transaction. For detailed information, see the following:
+     - [LIST &ndash; Attributes](../UserInterface/05_ListCancellationsChargebacks.md#cancellations-and-chargebacks-–-attributes)
+     - [LIST &ndash; Logs](../UserInterface/04_ListRefunds.md#refunds-–-logs)
+4. Return to the LIST and decide how to proceed with the transaction.
+
+
+## Void cancellation and chargeback transaction
+
+You can void a cancellation and chargeback transaction, if you want to disable the transaction for the Actindo database. 
+> [Info] The payment service provider will not be informed about changing the status to "void". It is only done to clear the database.
+
+#### Prerequisites
+
+The status of a cancellation and chargeback transaction is **not** success. In case of success, the cancellation and chargeback has already been further processed and is available under the *Confirmations* menu entry. <!---Stefan, ist das richtig oder ist es unter Refund? --> 
```

Comment:
```suggestion
The status of a cancellation and chargeback transaction is not **Success**. In case of success, the cancellation and chargeback has already been further processed and is available under the *Confirmations* menu entry. <!---Stefan, ist das richtig oder ist es unter Refund? --> 
```

Reply:
done